cartoframes/data/enrichment/points_enrichment.py

```diff
@@ -0,0 +1,95 @@
+import uuid
+
+from . import bigquery_client
+from ...auth import get_default_credentials
+from collections import defaultdict
+
+
+_ENRICHMENT_ID = 'enrichment_id'
+
+# TODO: process column name in metadata, remove spaces and points
+
+
+def enrich_points(data, variables, data_geom_column='geometry', filters=dict()):
+
+    credentials = get_default_credentials()
```

Comment:
This method could receive a credentials instance too, adding an optional parameter `credentials`. We are doing this way in the rest of the classes, prioritizing the parameter one:
```
credentials = credentials or get_default_credentials()
```

Reply:
agree